templates/devshells/c/flake.nix

```diff
@@ -35,5 +34,13 @@
       devShells = {
         default = nickelDerivation;
       };
+      apps = {
+        regenerate-lockfile = {
+          type = "app";
+          program = pkgs.lib.getExe (buildLockFile {
```

Comment:
I'm not sure why, but this `getExe` seems to be the thing causing the CI failure.

Reply:
Gne, nevermind that one, it's Friday

Reply:
The actuall issue is

```suggestion
          program = pkgs.lib.getExe (nickel-nix.lib.${system}.buildLockFile {
```

Reply:
It's already imported in let block above, so this shouldn't be the issue...